examples/subplots_axes_and_figures/axes_margins.py

```diff
@@ -1,6 +1,6 @@
 """
 =====================================================================
-Zooming in and out using Axes.margins and the subject of "stickiness"
+Axes.margins and "stickiness"
 =====================================================================
```

Comment:
Please adapt the length of the heading markup to the heading text. - For my aesthetic well-being. :smile:

Reply:
Ahh..I should have done that I the first place :sweat_smile: